pybamm/parameters/lithium_ion_parameters.py

```diff
@@ -224,6 +224,73 @@ def _set_dimensional_parameters(self):
             "Initial concentration in electrolyte [mol.m-3]"
         )
 
+        # mechanical prapmeters
+        if isinstance(self, pybamm.particle_cracking.BaseCracking):
+            self.flag_mechanics = pybamm.Parameter("Flag mechanical effects")
```

Comment:
do you need this flag? I think you should be able to handle this by defining this different depending on the options. as in, these parameters are only set when mechanics is true, so the flag should always be 1?

Reply:
This is an option to decide whether to consider mehcanical effects on Li diffusion. I chose to put this in a parameter because didn't find a better way to define this kind of options.

Reply:
solved as the above option comment

Reply:
`self.flag_mechanics` can be removed